src/app/features/shared/utils/fraudPredictionPilot.ts

```diff
@@ -0,0 +1,10 @@
+// caseId from (legacy) BWV always contains an underscore
+const strippedCaseId = (caseId: String) => Number(caseId.replace("_", ""))
```

Comment:
`Number(caseId.replace("_", ""))` should be `parseInt(caseId.replaceAll("_", ""), 10)`